go.mod

```diff
@@ -1,114 +1,129 @@
 module github.com/kubernetes-sigs/aws-efs-csi-driver
 
 require (
-	github.com/aws/aws-sdk-go v1.31.2
-	github.com/container-storage-interface/spec v1.2.0
+	github.com/aws/aws-sdk-go v1.40.29
+	github.com/container-storage-interface/spec v1.5.0
 	github.com/golang/mock v1.4.4
 	github.com/kubernetes-csi/csi-test v1.1.1
 	github.com/mitchellh/go-ps v0.0.0-20170309133038-4fdf99ab2936
-	github.com/onsi/ginkgo v1.12.2
+	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
-	google.golang.org/grpc v1.26.0
-	k8s.io/api v0.18.10
-	k8s.io/apimachinery v0.18.10
-	k8s.io/client-go v0.18.10
+	google.golang.org/grpc v1.38.0
+	k8s.io/api v0.22.2
+	k8s.io/apimachinery v0.22.2
+	k8s.io/client-go v0.22.2
 	k8s.io/klog v1.0.0
-	k8s.io/kubernetes v1.18.10
-	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
+	k8s.io/kubernetes v1.22.2
```

Comment:
@bertinatto I think this actually looks OK. If I run `go mod tidy` it seems to work: v1.22.2 is a valid k8s version.

Reply:
Strange, I got v.1.23.10 here (see output of `go mod tidy` above). I probably checked out the wrong branch.